Polyculture Use Case: Incomplete Groups (#165)
ElektraInitiative/PermaplanT: doc/usecases/polyculture/incomplete_groups.md

# Use Case: Incomplete Groups

## Summary

- **Scope:** Plants Layer
- **Level:** User Goal
- **Actors:** App User
- **Brief:** The user is informed about what plant groups need more planning.
- **Status:** Draft

> 

Reviewer: Please assign yourself.

## Scenarios

- **Precondition:**
  - The user has opened the app
  - The plants layer is visible
  - At least one plant is placed

>   - At least one of the placed plants is in need of another companion plant.

Reviewer: ```suggestion
  - At least to one of the placed plants a companion plant can be added.
```

- **Main success scenario:**

>   - Groups which need more companions are visually highlighted.

Reviewer: ```suggestion
  - Groups, where further companions are available, are visually highlighted.
```

> - **Alternative scenario:**

Reviewer: If there are no groups, then it would be a nice to have to indicate that all plants are happy. This could be also part of gamification. (@ThorbenStaufer)
